01-worker.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/bash
+set -o nounset -o errexit
+
+echo Environment=KUBELET_EXTRA_ARGS=--node-ip=${NODE_PRIVATE_IP} >> /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+
+systemctl daemon-reload
```

Comment:
`systemctl restart kubelet` could be needed, just in case. I have not checked does `kubeadm init` does restart though.